examples/custom_searcher_example.py

```diff
@@ -0,0 +1,41 @@
+"""
+An example using a Searcher (HEBOSearch) not defined
+directly in TuneSearchCV.
+"""
+
+from tune_sklearn import TuneSearchCV
+from ray import tune
+from ray.tune.suggest.hebo import HEBOSearch
+from sklearn.ensemble import RandomForestClassifier
+from sklearn import datasets
+from sklearn.model_selection import train_test_split
+import numpy as np
+
+digits = datasets.load_digits()
+x = digits.data
+y = digits.target
+x_train, x_test, y_train, y_test = train_test_split(x, y, test_size=.2)
+
+clf = RandomForestClassifier()
+param_distributions = {
+    "n_estimators": tune.randint(20, 80),
+    "max_depth": tune.randint(2, 10)
+}
+
+# If a Searcher is initialized without specifying search space,
+# it will use the space passed to TuneSearchCV
+# (in this example: param_distributions)
+searcher = HEBOSearch()
+
+# It is also possible to use user-defined Searchers, as long as
+# they inherit from ray.tune.suggest.Searcher and have the following
+# attributes: _space, _metric, _mode
+
+tune_search = TuneSearchCV(
+    clf, param_distributions, n_trials=3, search_optimization=searcher)
```

Comment:
so in this case you don't need to specify metric/mode (because it's automatic)?

Reply:
If metric, mode and space are unspecified in the Searcher, then the ones defined in TuneSearchCV will be used (and it has its default ones)